bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,358 @@
+
+The above command will:
+1. Generate a new BLS key pair
+2. Associate it with your validator address
+3. Store it in your node's configuration file at 
+`~/.<path>/config/priv_validator_key.json` alongside your validator key
+
+This key will be used automatically by your validator software when it needs 
+to participate in epoch-end signature collection. The BLS signatures help 
+create compact, efficient proofs of consensus that can be later timestamped to 
+Bitcoin.
+
+> ⚠️ **Important**: The `priv_validator_key.json` file contains sensitive key 
+> material. Make sure to backup this file and store it securely, as it's 
+> essential for your validator's operation and cannot be recovered if lost.
+> Note that BLS keys are currently not supported by Horcrux or other remote signing 
+> solutions, so alternative security measures should be considered for key 
+> management such as hardware wallets or offline solutions.
+
+## 4. CometBFT Validator Configuration
+
+First, retrieve your validator's consensus public key using the following 
+command: 
+
+```shell
+babylond tendermint show-validator --home <home>
+```
+
+This command reads your validator's key information from 
+`priv_validator_key.json` and outputs only the public key in a specific format 
+required for validator registration. The output will look like:
+
+```shell
+{"@type":"/cosmos.crypto.ed25519.PubKey","key":"0Wlt7ZPl0uvv7onsw4gP8FSQJUk986zMcOdWselDPM4="}
+```
+
+You'll need this formatted public key output to create your validator's 
+configuration file in the next step.
+
+Now we can use the output of the command above and replace the `pubkey` and
+`<home>` value in the example below. Subsequently run the following command to 
+create the validator configuration file:
+
+```shell
+cat > <home>/config/validator.json << EOF
+{
+  "pubkey": {"@type":"/cosmos.crypto.ed25519.PubKey","key":"0Wlt7ZPl0uvv7onsw4gP8FSQJUk986zMcOdWselDPM4="},
+  "amount": "1000000ubbn",
+  "moniker": "my-validator",
+  "commission-rate": "0.10",
+  "commission-max-rate": "0.20",
+  "commission-max-change-rate": "0.01",
+  "min-self-delegation": "1"
+}
+EOF
+```
+
+Parameters:
+- `pubkey`: Your validator's public key (the output you received before)
+- `amount`: Initial self-delegation amount
+- `moniker`: Your validator's name/identifier
+- `commission-rate`: Validator commission rate 
+- `commission-max-rate`: Specifies the maximum you can raise your commission in the future.
+- `commission-max-change-rate`: Maximum daily commission change rate
+- `min-self-delegation`: Minimum amount you must keep self-delegated
+
+If you prefer to add this manually or are having issues, another option is to 
+create a `validator.json` file manually and then paste the above json into it 
+but remember to replace all values with the actual values you want to use.
+
+## 5. Creating a Validator
+
+> ⚠️ **Important**: You will need a funded account for this step
```

Comment:
How about we move this note before the execution of the `create-validator` command? Makes more sense to have it closer to the command that requires the funds